pkg/controller/authentication/ingress.go

```diff
@@ -804,6 +810,13 @@ func platformOidcIngress(instance *operatorv1alpha1.Authentication, scheme *runt
 			},
 		},
 	}
+	if ingressUpdateRequired {
+		err = r.client.Update(context.TODO(), newIngress)
```

Comment:
where are you setting this variable? @rashmi43

Reply:
Wanted to ask you that